experimental/packages/opentelemetry-sdk-metrics-base/src/InstrumentDescriptor.ts

```diff
@@ -42,7 +42,7 @@ export function createInstrumentDescriptor(name: string, type: InstrumentType, o
     name,
     type,
     description: options?.description ?? '',
-    unit: options?.unit ?? '1',
+    unit: options?.unit === '' ? '1' : options?.unit ?? '',
```

Comment:
If I interpret the spec correctly, the spec says that null should be the same as an empty string. As there are no words on the default value `"1"`, I believe this coercion to `"1"` should be removed.

> If the unit is not provided or the unit is null, the API and SDK MUST make sure that the behavior is the same as an empty unit string.

Reply:
Cool, I can update it to default to an empty string instead.

Any concerns on that @pichlermarc or @dyladan?